>From 74e8ec86d53b1f427757c4b1eed917e43cf325d6 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Mon, 18 May 2015 00:36:37 +0900
Subject: [PATCH] hack: avoid annoying checkout after clone

After exec debcheckout, if you want to do "gbp import-orig" it would fail
since upstream branch is not checkouted (similar issue to pristine-tar branch,
too).
---
 scripts/debcheckout.pl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/debcheckout.pl b/scripts/debcheckout.pl
index c5fdd8a..b5bd35e 100755
--- a/scripts/debcheckout.pl
+++ b/scripts/debcheckout.pl
@@ -1138,6 +1138,7 @@ EOF
 		$cmd .= " && git branch --track $head remotes/origin/$head";
 		system($cmd);
 	    }
+	    system("cd $wcdir && (git checkout upstream; git checkout pristine-tar; git checkout master) >/dev/null 2>&1");
 	}
     } elsif ($repo_type eq 'hg') {
 	my $username = '';
-- 
2.1.4

